src/status_im/ui/screens/profile/user/views.cljs

```diff
@@ -25,7 +25,7 @@
 
 (defn my-profile-toolbar []
   [toolbar/toolbar {}
-   nil
+   [toolbar/nav-back-close-app]
```

Comment:
why do we need this?

Reply:
ok i understand, they meant only physical back button on android devices , we don't need to show back button in toolbar

Reply:
https://github.com/status-im/status-react/blob/develop/src/status_im/android/core.cljs#L21

Reply:
https://github.com/status-im/status-react/blob/develop/src/status_im/android/core.cljs#L29

Reply:
@kwingram25 so it shold work, we don't need changes for this requirements, could you please revert `nav-back-close-app`, thanks, sorry for the confusion

Reply:
ok it will not work, we need to add exit app case